content/debugging/docs/connect-spark-ui.md

```diff
@@ -0,0 +1,69 @@
+# **Connect to Spark UI running on the Driver Pod**
+
+To obtain more detail about their application or monitor their job execution, Spark application developers can connect to Spark-UI running on the Driver Pod.
+
+This page shows how to use `kubectl port-forward` to connect to the Job's Driver Pod running in a Kubernetes cluster. This type of connection is useful for debugging purposes.
+
+### Submitting the job to a virtual cluster
+
+**Request** 
+```
+cat >spark-python.json << EOF
+{
+  "name": "spark-python-in-s3", 
+  "virtualClusterId": "<virtual-cluster-id>", 
+  "executionRoleArn": "<execution-role-arn>", 
+  "releaseLabel": "emr-6.3.0-latest", 
+  "jobDriver": {
+    "sparkSubmitJobDriver": {
+      "entryPoint": "s3://<s3 prefix>/trip-count.py", 
+       "sparkSubmitParameters": "--conf spark.driver.cores=4  --conf spark.executor.memory=20G --conf spark.driver.memory=20G --conf spark.executor.cores=4"
+    }
+  }, 
+  "configurationOverrides": {
+    "applicationConfiguration": [
+      {
+        "classification": "spark-defaults", 
+        "properties": {
+            
+         }
+      }
+    ], 
+    "monitoringConfiguration": {
+      "cloudWatchMonitoringConfiguration": {
+        "logGroupName": "/emr-containers/jobs", 
+        "logStreamNamePrefix": "demo"
+      }, 
+      "s3MonitoringConfiguration": {
+        "logUri": "s3://joblogs"
+      }
+    }
+  }
+}
+EOF
+aws emr-containers start-job-run --cli-input-json file:///spark-python.json
+```
+
+Once the job is submitted successfully, run `kubectl get pods -n <virtual-cluster-k8s-namespace> -w ` command to watch all the pods, until you observe the driver pod is in the "Running" state. The Driver pod's name usually is in `spark-<job-id>-driver` format.
```

Comment:
Could you add a pre-requisite section:
* AWS cli should be installed
* kubectl should be installed
* Customer should run `aws eks update-kubeconfig --name <cluster--name> --region <us-west-2> to download kubeconfig file and use currect context to talk to API server.